src/main/java/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
```

Comment:
The package name 'dao' is not quite descriptive of its contents. Consider renaming it to 'io' or 'filereader' to better reflect the responsibility of the classes within.